README.md

```diff
@@ -96,6 +96,7 @@ Example:
 | `FUTURE_MONTHS`                       | `1`                | The amount of months that will be scrapped in the future, starting from the day calculated using `DAYS_BACK`                                                              |
 | `TRANSACTION_HASH_TYPE`               | ``                 | The hash type to use for the transaction hash. Can be `moneyman` or empty. The default will be changed to `moneyman` in the upcoming versions                             |
 | `ADDITIONAL_TRANSACTION_INFO_ENABLED` | `'false'`          | If set to `'true'`, enables the `additionalTransactionInformation` option in the underlying scraper, which may provide more detailed transaction data for some providers. |
+| `RAW_TRANSACTION_DATA_DISABLED`       | `'false'`          | If set to `'true'`, disables the raw TransactionRow JSON data column in Google Sheets. By default (`'false'`), the raw column is included.                                |
```

Comment:
I have a better idea - let's remove the env var that disables it. Instead, use the existence of the raw column as the gate to control it.